com.note.slayer.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.noteslayer.app
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: noteslayer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+separate-locales: false
+rename-desktop-file: noteslayer.desktop
+rename-icon: noteslayer
+modules:
+  - name: noteslayer
+    buildsystem: simple
+
+    sources:
+      - type: file
+        url: https://github.com/X-SLAYER/note_slayer_public/releases/download/V0.1/note_slayer_linux.deb
+        sha256: 5aee6f197601b82b65355dad919477789b466a27ae7c8bbd33d19a90d0d01270
+        dest-filename: note_slayer_linux.deb
+      - type: script
+        dest-filename: noteslayer.sh
+        commands:
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - exec zypak-wrapper /app/noteslayer/noteslayer "$@"
+
+    build-commands:
+      - ar x note_slayer_linux.deb
+      - rm -f note_slayer_linux.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r opt/* usr/* $FLATPAK_DEST
+      - rm -rf opt usr
+      - install noteslayer.sh $FLATPAK_DEST/bin/noteslayer
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.appdata.xml
```

Comment:
I doubt that this is still valid since now `FLATPAK_ID` has a different value but the source archive hasn't changed.